.goreleaser-stable.yaml

```diff
@@ -66,6 +67,16 @@ publishers:
     cmd: curl -F package=@{{ .ArtifactName }} https://{{ .Env.FURY_TOKEN }}@push.fury.io/{{ .Env.FURY_USER }}/
 aurs:
   - homepage: https://kraftkit.sh
+    ids:
+#@ targets = {
+#@   "linux-amd64": {"os": "linux", "arch": "amd64"},
+#@   "linux-arm64": {"os": "linux", "arch": "arm64"}
+#@ }
+#@ for binary in ["kraft"]:
+#@ for target, specs in targets.items():
```

Comment:
`specs` looks like it is not used? the list of targets could be simplified to just:
```
#@ targets = [
#@   "linux-amd64",
#@   "linux-arm64",
#@ ]
```
?